css.css

```diff
@@ -0,0 +1,486 @@
+    margin-top: -50px;
+    margin-left: 200px;   
+    cursor: pointer;
+}
+nav a {
+    padding-left: 30px;
+    text-decoration: none;
+    color: white;
+}
+nav a:hover {
+    background-color: #046e8f;
+    transition: 0.3s linear;
+    text-decoration: none;
+}
+nav p {
+    margin-top: 50px;
+    color: red;
+}
+.main-block {
+    width: 1100px;
+    height: 1400px;
+    margin-left: 50px;
+    /*float: right;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.left-block {
+    
+    width: 550px;
+    height: 1400px;
+    float: left;
+}
+.right-block {
+    
+    width: 550px;
+    height: 1400px;
+    float: left;
+}
+menu {
+    width: 1400px;
+    height: 2000px;
+}
+.about-me {
+    width: 640px;
+    height: 608px;
+    background: #046e8f;
+    float: left;
+    padding-left: 50px;
+}
+.latest-articles {
+    width: 640px;
+    height: 700px;
+    float: left;
+    padding-left: 50px;
+}
+.my-photo {
+    width: 600px;
+    height: 600px;
+    background-image: url("./assets/k.jpg");
+    float: left;
+    padding-top: 8px;
+}
+.mini-block-photo {
+    height: 370px;
+    width: 340px;
+    margin-left: 15px;
+    display: inline-block; /* Строчно-блочный элемент */
+    overflow: hidden;
+}
+.mini-block-photo:hover {
+    cursor: pointer;
+}
+.mini-block {
+    margin-top: 70px;
+    width: 400px;
+    height: 520px;
+    background: #093e52;
+    margin-left: 20px;
+    padding-left: 30px;
+    padding-top: 30px;
+    
+}
+.mini-block h1 {
+    font-size: 20px;
+    color: #38AECC;
+}
+.mini-block h2 {
+    font-size: 12px;
+    color: rgb(223, 223, 223);
+    line-height: 1.5;
+    font-weight: 200;
+}
+.button {
+    height: 40px;
+    width: 200px;
+    margin-top: 50px;
+    margin-left: 20px;
+    border-radius: 35px;
+    border: 2px solid #38AECC;
+    text-align: center;
+    padding-top: 20px;
+    color: white;
+}
+.button:hover {
+    height: 40px;
+    width: 200px;
+    margin-left: 20px;
+    background: #38AECC;
+    margin-top: 50px;
+    border-radius: 35px;
+    cursor: pointer;
+    transition: 0.5s linear;
+    text-align: center;
+    padding-top: 20px;
+    color: white;
+}
+a:hover {
+    cursor: pointer;
+}
+.my-info {
+    width: 1000px;
+    height: 500px;
+    float: left;
+    padding-left: 20px;
+}
+.my-info h1 {
+    margin-top: 40px;
+    margin-left: 50px;
+    font-size: 40px;
+    color: #38AECC;
+}
+.my-info h2 {
+    margin-top: 40px;
+    margin-left: 50px;
+    font-size: 18px;
+    color: white;
+    font-weight: 200;
+    line-height: 1.5;
+}
+.my-info h3 {
+    margin-top: 40px;
+    margin-left: 50px;
+    font-size: 20px;
+    color: white;
+    font-weight: 200;
+    line-height: 1.5;
+}
+.button1 {
+    padding-top: 20px;
+    height: 40px;
+    width: 200px;
+    margin-top: 50px;
+    margin-left: 50px;
+    border-radius: 35px;
+    color: white;
+    border: 2px solid #38AECC;
+    text-align: center;
+}
+.button1:hover {
```

Comment:
make all selector names **readable**
`button1` is not good name for selector. better name it like `portfolio-button`...